packages/reaction-i18n/private/data/i18n/en.json

```diff
@@ -93,7 +93,9 @@
           "inventorySettingsLabel": "Inventory Settings",
           "analyticsSettingsLabel": "Analytics Settings",
           "paypalSettingsLabel": "PayPal Settings",
-          "genericPaymentSettingsLabel": "Generic Payment Settings"
+          "genericPaymentSettingsLabel": "Generic Payment Settings",
```

Comment:
I think that "Example Payment" was supposed to replace "Generic Payment" so there shouldn't be both?